app/managers.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name_: str, last_name_: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name}"
+            " (first_name, last_name) VALUES (?, ?)",
+            (first_name_, last_name_))
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}")
```

Comment:
same + It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.